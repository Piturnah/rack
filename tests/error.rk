1 if 0 print