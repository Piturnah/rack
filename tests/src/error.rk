fn main in 
  1 if 0 print end
end
