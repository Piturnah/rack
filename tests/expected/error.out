----STDOUT----
0

----STDERR----
